Breadcrumbs.py

```diff
@@ -108,4 +132,213 @@ def get_points_by_row(row):
               breadcrumbs[index_of_breadcrumb] = breadcrumbs[index_of_breadcrumb][0:length_of_trim]
           break
       number_of_characters_left -= current_length
-    view.set_status('breadcrumbs',separator.join(breadcrumbs+['']))
+  return breadcrumbs
+
+
+def copy(view, text):
+  sublime.set_clipboard(text)
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+class BreadcrumbsEventListener(sublime_plugin.ViewEventListener):
```

Comment:
> I expect that having phantoms follow your cursor is going to be terrible. The lines are moved apart to make space for a phantom, so if you move your cursor down a line that could be jumpy as the phantom is removed and then recreated a line down.

True. I haven't think about that.

> The popup seems like it could be less intrusive, although it will also follow your cursor horizontally and that's super annoying too.

The spec says:
> The default location of -1 will display the popup at the cursor, otherwise a text point should be passed.

so perhaps it is possible to pass a location with column fixed to 0? 

>  I guess you'll have to try it and see.

Sure! I've already tried a few combinations (a block phantom below the current line, a block phantom above current line, a popup bellow current line aligned to column 0, etc.) and indeed all of them are annoying to me. The block phantom below current line is least annoying, but also seems to me difficult to read, as I intuitively seek information about scope above my cursor, not below). 

So perhaps we will need something more complicated, to make it feel less intrusive.
One idea would be to show the breadcrumbs at the top of the window, and only show breadcrumbs related to the scopes opening lines of which are not directly visible in the screen.

Here is a sketch of this idea:
![idea](https://user-images.githubusercontent.com/133093/29428032-07a7e9a0-838c-11e7-9539-6dd0ade884c9.png)
 
This should directly address my original problem which was "I have no idea where am I because the scope was opened above my current field of view". Also it could make more sense to display them vertically, with the left edge of each breadcrumb aligned to the column in which it originally appeared, so that you can use the same "part of the brain" that you normally use when checking which scope is responsible for which column.